src/track/beatgrid.h

```diff
@@ -89,6 +93,8 @@ class BeatGrid final : public Beats {
     mixxx::track::io::BeatGrid m_grid;
     // The length of a beat in samples
     double m_dBeatLength;
+    const int c_beatsPerBar = 4;
```

Comment:
Define `constexpr` constants for these default values instead of using const members. As long as the constants do not need to be publicly visible enclose them in an anonymous namespace in the .cpp file.

Reply:
IMHO this should be a control object. This would make it possible to modify them via controller buttons (maybe PARAMETER +/- ?).

Reply:
I like the suggestion but I would like to include that in a later PR if you don't mind.